From 570db52d7fd3b4deffe785e60fb362e1496844ee Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@grinn-global.com>
Date: Tue, 26 Sep 2017 12:10:53 +0200
Subject: [PATCH] http-parser: Fix typo in switch FALLTHROUGH comment

This also suppresses -Wimplicit-fallthrough warning, which is enabled
in gcc7 by default.

PR for upstream: https://github.com/kernelsauce/turbo/pull/323

Signed-off-by: Marcin Niestroj <m.niestroj@grinn-global.com>
---
 deps/http-parser/http_parser.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/http-parser/http_parser.c b/deps/http-parser/http_parser.c
index 8cf6fb0..2ff1073 100644
--- a/deps/http-parser/http_parser.c
+++ b/deps/http-parser/http_parser.c
@@ -2095,7 +2095,7 @@ http_parser_parse_url(
       case s_req_server_with_at:
         found_at = 1;
 
-      /* FALLTROUGH */
+      /* FALLTHROUGH */
       case s_req_server:
         uf = UF_HOST;
         break;
-- 
2.14.1

